stable/distribution/ci/test-values.yaml

```diff
@@ -1,31 +1,22 @@
 # CI values for Distribution
 
-# replicaCount: 2
-# rbac:
-#   create: false
-#
-# serviceAccount:
-#   create: false
+replicaCount: 2
```

Comment:
Are we doing any functional test with 2 replicas? If not, I suggest leaving this on 1 to save time.